src/react/components/pages/homepage/homePage.tsx

```diff
@@ -139,11 +170,31 @@ export default class HomePage extends React.Component<IHomepageProps> {
     }
 
     private loadSelectedProject = async (project: IProject) => {
-        await this.props.actions.loadProject(project);
-        this.props.history.push(`/projects/${project.id}/edit`);
+        console.log(project);
+        if (project.version === "v1-to-v2") {
+            console.log("loadingSelectedV1Project!!");
+            await this.settingsConfirm.current.open(project);
+            await this.props.actions.loadProject(project);
+            this.props.history.push(`/projects/${project.id}/settings`);
+        } else {
+            await this.props.actions.loadProject(project);
+            this.props.history.push(`/projects/${project.id}/edit`);
+        }
     }
 
     private deleteProject = async (project: IProject) => {
         await this.props.actions.deleteProject(project);
     }
+
+    private convertProject = async (project: any) => {
+        const importService = new ImportService();
+        let projectJson;
+        try {
+            projectJson = await importService.convertV1(project);
+        } catch (e) {
+            throw new AppError(ErrorCode.ProjectUploadError, "Error uploading v1 project file");
```

Comment:
Please create a new `ErrorCode` for this condition since it's very different from an project upload error.